src/components/Breadcrumb/breadcrumb-utils.js

```diff
@@ -0,0 +1,3 @@
+import { breadcrumbStore } from '@/stores';
+
+export const set = value => breadcrumbStore.set(value);
```

Comment:
Let's keep utils as pure functions only. This causes a side effect and is basically a 1:1 callthrough to a breadcrumbStore-method invocation.

Reply:
Fixed.